server/app/models/block.rb

```diff
@@ -20,215 +21,109 @@ def contained_accounts
   end
 
   def cellbase_id
-    "0x#{super.unpack("H*").first}"
+    "#{ENV['DEFAULT_HASH_PREFIX']}#{super.unpack1('H*')}" if super.present?
   end
 
   def cellbase_id=(cellbase_id)
-    super([cellbase_id[2..-1]].pack("H*"))
+    cellbase_id = [cellbase_id.delete_prefix(ENV["DEFAULT_HASH_PREFIX"])].pack("H*") if cellbase_id.present?
+    super
   end
 
   def difficulty
-    "0x#{super.unpack("H*").first}"
+    "#{ENV['DEFAULT_HASH_PREFIX']}#{super.unpack1('H*')}" if super.present?
   end
 
   def difficulty=(difficulty)
-    super([difficulty[2..-1]].pack("H*"))
+    difficulty = [difficulty.delete_prefix(ENV["DEFAULT_HASH_PREFIX"])].pack("H*") if difficulty.present?
+    super
   end
 
   def block_hash
-    "0x#{super.unpack("H*").first}"
+    "#{ENV['DEFAULT_HASH_PREFIX']}#{super.unpack1('H*')}" if super.present?
   end
 
   def block_hash=(block_hash)
-    super([block_hash[2..-1]].pack("H*"))
+    block_hash = [block_hash.delete_prefix(ENV["DEFAULT_HASH_PREFIX"])].pack("H*") if block_hash.present?
+    super
   end
 
   def parent_hash
-    "0x#{super.unpack("H*").first}"
+    "#{ENV['DEFAULT_HASH_PREFIX']}#{super.unpack1('H*')}" if super.present?
   end
 
   def parent_hash=(parent_hash)
-    super([parent_hash[2..-1]].pack("H*"))
+    parent_hash = [parent_hash.delete_prefix(ENV["DEFAULT_HASH_PREFIX"])].pack("H*") if parent_hash.present?
```

Comment:
We have too many property setting/getting with `delete_prefix(ENV["DEFAULT_HASH_PREFIX"])`, `pack("H*")`, etc. Wondering if there's a better way (mapping or wrapping) to make this more clear with less duplication.

Reply:
Thank you, this is a WIP PR,  When all is done, I will refactor this。Sorry, I submitted PR incorrectly yesterday.